File: Programming Basics/JavaScript/Introduction/Exercises/fish-tank.js
function solve(input) {
    const length = Number(input[0]);
    const width = Number(input[1]);
    const height = Number(input[2]);
    const percentage = Number(input[3]);

    const volume = 0.001 * length * width * height;
    const freeSpace = volume * (1 - 0.01 * percentage);
    console.log(freeSpace);
}

File: Programming Basics/C++/Introduction/Exercises/FishTank/FishTank.cpp
#include <iostream>

int main()
{
    int length, width, height;
    double percentage;
    std::cin >> length >> width >> height >> percentage;

    double volume = 0.001 * length * width * height;
    double freeSpace = volume * (1 - 0.01 * percentage);
    std::cout << freeSpace << std::endl;
}